jsonforms-editor/src/editor/components/EditorPanel.tsx

```diff
@@ -27,7 +27,7 @@ export interface EditorTab {
 }
 
 interface EditorPanelProps {
-  editorTabs?: EditorTab[];
+  editorTabs?: EditorTab[] | null;
```

Comment:
Is it needed here? we don't separate between default and no tabs here, do we?

Reply:
No, we don't, I changed too much. I reverted the changes. The types in the `JsonFormsEditorProps` already contain `null`.